Sources/XCTest/XCTAssert.swift

```diff
@@ -52,25 +52,19 @@ private enum _XCTAssertionResult {
     case UnexpectedFailure(ErrorProtocol)
 
     var expected: Bool {
-        switch (self) {
-        case .UnexpectedFailure(_):
-            return false
-        default:
-            return true
+        switch self {
+        case .UnexpectedFailure(_): return false
```

Comment:
Cool.
Just as a side-question: Is there way to make a commit that affects multiple files? I'm editing the code online, so when I try to switch to another file, it would not save the changes from the current one.